src/main/java/ai/rev/ErrorInterceptor.java

```diff
@@ -1,10 +1,16 @@
-package revai;
+package ai.rev;
```

Comment:
if we are going to be doing this, I think we should have another top-level package, e.g. `ai.rev.speechtotext`. Since now we are using `ai.rev` as basically the company domain name prefix, and i don't think putting all the classes directly into that makes sense, if we ever have other APIs or libraries we could easily have name collisions.

Reply:
shoot I didn't think of that... I'll reach out to Sonatype and see if we can add that to our group id. I imagine we can because we now own that name space in the central repo. Do we want `ai.rev.speechtotext`?

Reply:
I'm going to make the change. I heard back from Sonatype and they grant permissions at the top level group id so it won't be an issue for this project.

Reply:
cool, yeah `ai.rev.speechtotext` sounds good.